polaris-react/src/components/Badge/index.ts

```diff
@@ -1 +1,3 @@
 export * from './Badge';
+export * from './components/Pip';
```

Comment:
If you really want to keep it a sub-component of Badge, I think exposing it via `Badge.Pip = Pip;` is enough, we shouldn’t export it here:
```suggestion
```